packages/indicators/src/builders/buildArithmetic.ts

```diff
@@ -62,7 +78,12 @@ export const buildArithmetic: Builder = async input => {
   const { aggregator, config: configInput, fetchOptions } = input;
   const config = await validateConfig<ArithmeticConfig>(configInput, configValidators);
```

Comment:
@Alastair-L  @edmofro Do you think it's worth adding validation for the `defaultValues` field? It should be

* either "not present", or
* an object with all keys included in the formula as data elements, and numeric values

By having a look at the code, it seems that wrong defatulValues keys will not result in unwanted behaviour:
```js
{
  formula: 'A + B',
  defaultValues: { A: 0, B: 0, C: 0 } // 'C' will eventually be ignored in `runArithmetic`
}
```

But may still be worth it given the existing validation is pretty thorough

Reply:
Good idea, I wasn't sure exactly how to do that, see below for my attempt.